deployment/serverless.yml

```diff
@@ -19,6 +19,14 @@ provider:
     Product: ${env:PROJECT_GROUP_SHORT}
     Programme: ${env:PROGRAMME}
     Service: ${env:PROJECT_GROUP_SHORT}
+  apiGateway:
+    apiKeySourceType: HEADER
+    apiKeys:
+      - name: NHS_UK_API_KEY
+        value: ${self:custom.secret.NHS_UK_API_KEY}
+custom:
+  secret: ${ssm:/aws/reference/secretsmanager/uec-dos-int-dev-api-key}
```

Comment:
Can the secret key name and key be variables and stored in task.mk (build/automation/var/profile/task.mk)

Reply:
done